### ISSUE TITLE
Re-arrange RPC arguments to allow route agnosticism

### ISSUE THREAD

**Issue by faultyserver** [propershark/timetable_cpp#5]
In many cases, queries for the next arrivals at a station are interested in _all_ arrivals, not just those from a particular route. However, the current RPC argument structure for the `visits_*` family is `(station, route, timestamp, count)`, meaning `route` is not a parameter that can be omitted.

To allow for both calls that request arrivals from a particular route and that don't, a better RPC definition is:

```c++
visits_after(station, timestamp, route="", count);
```

Note that the `route` parameter is optional, but is not the last parameter. The current WAMP library that timetable uses does not allow for optional parameters, nor does it allow registering multiple procedures under the same name. In other words, this structure is not possible with the current WAMP library, but is the ideal case and should be considered if the underlying WAMP library changes.

For now, however, a simple fix is to create two distinct procedures that take these different argument sets. They could be:

```c++
visits_after(station, timestamp, count);
visits_after_from_route(station, timestamp, route, count);
```

This should handle the structure for now, but should not be considered a final solution.

**Comment by faultyserver** [comment]
This is directly related to #3 as the gateway to implementing RPCS with optional route parameters.

**Comment by faultyserver** [comment]
Closed by #7.
